pilot/helpers/files.py

```diff
@@ -50,26 +50,35 @@ def get_file_contents(
     will be a Python string. If that fails, it will be treated as a
     binary file and `content` will be a Python bytes object.
     """
+    # Normalize the path to avoid issues with different path separators
+    full_path = os.path.normpath(os.path.join(project_root_path, path))
+
+    # Check if the path is a file
+    if not os.path.isfile(full_path):
```

Comment:
It'd be better to do this as a general `except Exception` below so it can catch any other errors that might happen (eg. permission error, etc).